common/changes/@microsoft/rush/sennyeya-shard-schema-updates_2024-09-13-22-46.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "(DEPRECATED) Deprecated the `sharding.shardOperationSettings` property in the project `config/rush-project.json` in favor of an `operationSettings` entry for an operation with a suffix of `:shard`.",
```

```suggestion
      "comment": "Deprecate the `sharding.shardOperationSettings` property in the project `config/rush-project.json` in favor of an `operationSettings` entry for an operation with a suffix of `:shard`.",
```

I don't think we have other instances of "(DEPRECATED)" in the changelog.

It'd also be a good idea to update https://github.com/microsoft/rushstack/blob/main/apps/rush/UPGRADING.md with instructions. Note that this will go out in the next minor bump (so, 5.135.0)